comparisontool/management/commands/load_bah.py

```diff
@@ -8,16 +10,18 @@
 
 
 class Command(BaseCommand):
-    args = '<path>'
     help = 'Load the specified CSV into the BAH database'
 
+    def add_arguments(self, parser):
+        parser.add_argument('path_spec')
+
     def handle(self, *args, **options):
-        for path_spec in args:
-            for path in glob(path_spec):
-                print "loading %s" % path
-                with open(path) as csv_file:
-                    reader = csv.DictReader(csv_file)
-                    BAHRate.objects.all().delete()
-                    for record in reader:
-                        new_rate = BAHRate(zip5=record['ZIP'], value=record['BAH'])
-                        new_rate.save()
+        path_spec = options['path_spec']
+        for path in glob(path_spec):
+            print("loading BAH file:", path)
```

Comment:
I just remembered something. No need to change this if you feel it's out of scope, but IIRC Django management commands have a preferred way of writing to stdout, see [here](https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/#management-commands-output).

@chosak may have thoughts too; it's been a while since I wrote a Django management command, I'm a bit rusty.

Reply:
I agree that using `self.stdout.write` is somewhat more preferable.